src/components/basic/Button/button.tsx

```diff
@@ -239,6 +239,22 @@ export const ClaimButton = styled(PrimaryOutlinedButton)`
   }
 `;
 
+export const TwitterButton = styled(PrimaryOutlinedButton)`
```

Comment:
Instead of styled components, we are currently using tailwind and index.scss file to style elements. Please change your TwitterButton implementation.